apps/prairielearn/src/components/DeleteCourseInstanceModal.tsx

```diff
@@ -0,0 +1,194 @@
+import { useEffect, useRef, useState } from 'preact/hooks';
+
+import type { CourseInstance } from '../lib/db-types.js';
+
+export function DeleteCourseInstanceModal({
+  shortName,
+  enrolledCount,
+  csrfToken,
+}: {
+  shortName: CourseInstance['short_name'];
```

Comment:
```suggestion
  shortName: StaffCourseInstanceContext['course_instance']['short_name'];
```

Ensures that it is non-nullable, which should be the case on this page AFAIK. Alteratively, just `string`.

Reply:
I switched to plain `string` in d6b24e3.